src/components/RightSidebar/RightSidebar.vue

```diff
@@ -22,9 +22,9 @@
 			<NcCounterBubble v-if="unreadMessagesCounter > 0"
 				class="chat-button-unread-messages-counter"
 				:class="{ 'chat-button-unread-messages-counter--highlighted': hasUnreadMentions }"
-				:type="hasUnreadMentions ? 'highlighted' : 'outlined'">
-				{{ unreadMessagesCounter }}
-			</NcCounterBubble>
+				:type="hasUnreadMentions ? 'highlighted' : 'outlined'"
+				:count="unreadMessagesCounter" />
+			<!-- FIXME looks weird -->
```

Comment:
?

Reply:
whoops

Reply:
As per discussion with design team, will do that